### PR TITLE
Horrible Queries Code

### PR DESCRIPTION
**Pull request by brebenelmihnea**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: content/5_Plat/RURQ.mdx [hunk 1]
```diff
@@ -37,15 +37,77 @@ queries.
...
-	/>
-</Resources>
+<LanguageSection>
+<CPPSection>
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+class fenwick_tree {
```
Comment (SansPapyrus683) [inline content/5_Plat/RURQ.mdx:47]: use the bit from the [purs module](https://usaco.guide/gold/PURS) for consistency

File: content/5_Plat/RURQ.mdx [hunk 2]
```diff
@@ -37,15 +37,77 @@ queries.
...
+		int sum = 0;
+		while (x > 0) {
+			sum += t[x];
+			x -= x & -x;
+		}
+		return sum;
+	}
+};
+
+void test_case() {
```
Comment (SansPapyrus683) [inline content/5_Plat/RURQ.mdx:73]: just put this in main

File: content/5_Plat/RURQ.mdx [hunk 3]
```diff
@@ -37,15 +37,77 @@ queries.
...
+	int n, q;
+	cin >> n >> q;
+
+	fenwick_tree bit(n), bit2(n);
```
Comment (SansPapyrus683) [inline content/5_Plat/RURQ.mdx:77]: perhaps more informative names than `bit` and `bit2` would be nice

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
nice

File: content/5_Plat/RURQ.mdx (hunk 1 above)
Comment (brebenelmihnea) [reply content/5_Plat/RURQ.mdx]: Done

File: content/5_Plat/RURQ.mdx (hunk 2 above)
Comment (brebenelmihnea) [reply content/5_Plat/RURQ.mdx]: Done

File: content/5_Plat/RURQ.mdx (hunk 3 above)
Comment (brebenelmihnea) [reply content/5_Plat/RURQ.mdx]: Done

**Review by SansPapyrus683 — APPROVED** [review]
does it still work

**Review by ryanchou-dev — APPROVED** [review]
great!

**Pull request merged**